$MirOS: ports/security/gnupg/patches/patch-mpi_mpi-internal_h,v 1.1 2008/04/01 15:47:02 tg Exp $
--- mpi/mpi-internal.h.orig	Thu Dec 20 17:22:02 2012
+++ mpi/mpi-internal.h	Wed Jan  2 21:23:57 2013
@@ -83,6 +83,9 @@ struct gcry_mpi {
 typedef mpi_limb_t *mpi_ptr_t; /* pointer to a limb */
 typedef int mpi_size_t;        /* (must be a signed type) */
 
+#undef ABS
+#undef MIN
+#undef MAX
 #define ABS(x) (x >= 0 ? x : -x)
 #define MIN(l,o) ((l) < (o) ? (l) : (o))
 #define MAX(h,i) ((h) > (i) ? (h) : (i))
